src/backend.cpp

```diff
@@ -705,6 +728,25 @@ void BackEnd::trustCertificate()
     connectToEndpoint();
 }
 
+void BackEnd::removeRecentConnection(const QString &name)
+{
+    if (mLastServerHosts.contains(name)) {
+        mLastServerHosts.removeAll(name);
+
+        QSettings settings;
```

Comment:
The same source code is used in method saveServerHost(). It is best to write a separate method.

Reply:
Done.